src/main/java/org/spongepowered/api/world/extent/BlockVolume.java

```diff
@@ -39,7 +39,7 @@
      * @param position The position
      * @return The block
      */
-    BlockLoc getBlock(Vector3d position);
+    BlockState getBlock(Vector3i position);
```

Comment:
Because BlockLoc requires an Extent, but a BlockVolume is a super-interface of Extent. This makes BlockVolume essentially useless unless used with Extent. Using BlockState means that buffers can now implement this interface.